database/src/main/java/com/firebase/ui/database/FirebaseIndexListAdapter.java

```diff
@@ -54,6 +56,6 @@ public FirebaseIndexListAdapter(Activity activity,
                                     @LayoutRes int modelLayout,
                                     Query keyRef,
                                     Query dataRef) {
-        super(activity, modelClass, modelLayout, new FirebaseIndexArray(keyRef, dataRef));
+        super(activity, modelClass, modelLayout, new FirebaseIndexArray(keyRef, dataRef.getRef()));
```

Comment:
@samtstern Yeah, this is the deprecated adapter so we can't break backwards compatibility.